atrium-api/src/types/integer.rs

```diff
@@ -28,6 +29,16 @@ macro_rules! uint {
             }
         }
 
+        impl<const MAX: $primitive> FromStr for $lim<MAX> {
+            type Err = String;
+
+            fn from_str(src: &str) -> Result<Self, Self::Err> {
+                Self::new(
+                    src.parse::<$primitive>().map_err(|_| format!("value is not an integer"))?,
```

Comment:
I'm thinking `.map_err(|e| e.to_string())` would be fine, but what do you think?

Reply:
Yeah, that is probably a better idea :)